### PR TITLE
[Test] test with ReadNode ,WithCleanFdsCacheThreshold,WithMaxFdNumsInCache,WithSyncEnable

### PR DESCRIPTION
**Pull request by JingchenZhang**
add unit test

### CODE REVIEW

File: bptree_test.go
Comment (elliotchenzichang) [inline bptree_test.go:1]: Could you help solute the issues raised by the review dog?  Thanks!

**Review by elliotchenzichang — COMMENTED** [review]
General LGTM, and could you help fix the questions raised by the review dog? thanks!

**Comment by elliotchenzichang** [comment]
I merged this PR into master and apparently your PR has some conflicts with him's, could you help resolve it? Thanks!

File: bptree_test.go
Comment (JingchenZhang) [reply bptree_test.go:1]: ok

**Comment by elliotchenzichang** [comment]
Merged, Thanks for your contribution!

**Pull request merged**